Library/Formula/elm-format.rb

```diff
@@ -0,0 +1,35 @@
+require "language/haskell"
+
+class ElmFormat < Formula
+  include Language::Haskell::Cabal
+  desc "formats Elm source code based on the official Elm Style Guide"
+  homepage "https://github.com/avh4/elm-format"
+  url "https://github.com/avh4/elm-format/archive/0.2.0-alpha.tar.gz"
+  version "3"
```

Comment:
`0.2.0-alpha` is the third release. I was just following that scheme.

https://github.com/avh4/elm-format/releases